tts-service/web/delete.go

```diff
@@ -0,0 +1,24 @@
+package web
+
+import (
+	"net/http"
+)
+
+func onDeleteByIdRequest(h getOrDeleteHandling, w http.ResponseWriter, r *http.Request) {
+
+	//Invoke service
+	id, err := getId(h.pathPrefix, r)
+	if err != nil {
+		handleError(err, w, r)
+		return
+	}
+
+	serviceErr := h.service.Delete(id)
+
+	if serviceErr != nil {
+		handleError(convertError(serviceErr), w, r)
+	} else {
+		addJsonHeader(w)
```

Comment:
No need to add Json Header for "No Content" status code - after all there's no content :)